packages/grid/x-data-grid-premium/src/hooks/features/export/serializer/excelSerializer.ts

```diff
@@ -173,7 +172,7 @@ const defaultColumnsStyles = {
   [GRID_DATETIME_COL_DEF.type as string]: { numFmt: 'dd.mm.yyyy hh:mm' },
 };
 
-const serializeColumn = (column: GridStateColDef, columnsStyles: ColumnsStylesInterface) => {
+const serializeColumn = (column: GridColDef, columnsStyles: ColumnsStylesInterface) => {
```

Comment:
No need to use `GridStateColDef` here, since we don't use any properties specific to `GridStateColDef`, like `computedWidth`